layout.js

```diff
@@ -1,3 +1,29 @@
+
+/**
+ * Update Aug 13, by Rhett Anderson
+ * 
+ * Handle degenerate case of having only two bitmaps differently. This prevents the clouds from being split
```

Comment:
These are great comments; I think we should either move them to the Pull Request comments section or keep them more generalized. For example, other games across devkit won't understand the clouds reference.